README.md

```diff
@@ -6,7 +6,7 @@
 Please see: [DOCUMENTATION.md](DOCUMENTATION.md) or [PDF](https://pdfdoc-purpleidea.rhcloud.com/pdf/https://github.com/purpleidea/vagrant-builder/blob/master/DOCUMENTATION.md).
 
 ## Installation:
-Please read the [INSTALL](INSTALL) file for instructions on getting this installed.
+Please read the [Installation instructions](https://github.com/purpleidea/vagrant-builder/blob/master/DOCUMENTATION.md#setup) to get started with Vagrant Builder.
```

Comment:
Good patch thanks!

I have the horrible curse of noticing certain small details (nitpicking). Shouldn't it be "installation instructions" (lower case I) ... If so please squash down and resubmit.

Thanks!!